client/src/app/(asset)/metadata/[id]/content/json-editor.tsx

```diff
@@ -114,37 +138,51 @@ export default function JSONEditor({
       <Typography className="italic text-white/50">
         To edit a key, double-click it. Press Enter to save, or Esc to cancel.
       </Typography>
+      <Typography className="italic text-white/50">
+        To delete an element from string array. Set the value to an empty
+        string.
+      </Typography>
       <JsonEditor
         data={meta}
         showErrorMessages
         enableClipboard={false}
-        defaultValue={""}
+        defaultValue={"null"}
```

Comment:
```suggestion
```

Reply:
defaultValue's default value is already `null`

Reply:
yeah but i need a default string